### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by jamesodo** [comment]
pypy3 is breaking stuff again.

**Comment by RayeN** [comment]
despite we excluded it from tox file.  James any idea why we still check pypy3?

**Comment by RayeN** [comment]
And this is probably a bugfix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # [Changelog](https://github.com/yola/demands/releases)
 
+## 4.1.0
+
+* Add proper handling of empty next page in paginated result set.
```
Comment (RayeN) [inline CHANGELOG.md]: We should probably mention that now we support a new option for pagination.

**Comment by jamesodo** [comment]
pypy3 is listed as the toxenv in the config.
https://travis-ci.org/yola/demands/jobs/216376082/config

Not sure how to change that...

**Comment by jamesodo** [comment]
Found it. pypy3 is also in the `.travis.yml` https://github.com/yola/demands/blob/fd6b396aef9e4e80c842b326ef6b551c0014fe55/.travis.yml#L11

**Comment by RayeN** [comment]
Thanks, James.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+  PaginatedResults now accepts `next_key` option.
```
Comment (RayeN) [inline CHANGELOG.md]: missed `*`?

**Pull request merged**